src/Hungary/Mews.Fiscalizations.Hungary.Tests/InvoiceTests.cs

```diff
@@ -95,29 +95,29 @@ public async Task SendCorrectionForeignCompanyInvoiceSucceeds(string countryCode
                 taxpayerId: taxpayerNumber.GetOrNull()
             ).Success.Get();
             var invoiceNumber = InvoiceNumber.Create($"INVOICE-{Guid.NewGuid()}").Success.Get();
-            var sendInvoicesResult = await SendInvoices(receiver, invoiceNumber);
-            await AssertInvoices(sendInvoicesResult);
+            var sendInvoicesResult = await SendInvoices(receiver, invoiceNumber).ConfigureAwait(continueOnCapturedContext: false);
+            await AssertInvoices(sendInvoicesResult).ConfigureAwait(continueOnCapturedContext: false);
 
-            var sendModificationInvoiceResponse = await SendModificationInvoices(receiver, originalInvoiceNumber: invoiceNumber);
-            await AssertInvoices(sendModificationInvoiceResponse);
+            var sendModificationInvoiceResponse = await SendModificationInvoices(receiver, originalInvoiceNumber: invoiceNumber).ConfigureAwait(continueOnCapturedContext: false);
+            await AssertInvoices(sendModificationInvoiceResponse).ConfigureAwait(continueOnCapturedContext: false);
         }
 
         private async Task<ResponseResult<string, ResultErrorCode>> SendInvoices(Receiver receiver, InvoiceNumber invoiceNumber = null)
         {
-            var exchangeToken = await NavClient.GetExchangeTokenAsync();
+            var exchangeToken = await NavClient.GetExchangeTokenAsync().ConfigureAwait(continueOnCapturedContext: false);
             return await NavClient.SendInvoicesAsync(
                 token: exchangeToken.SuccessResult,
                 invoices: Sequence.FromPreordered(new[] { CreateInvoice(receiver, invoiceNumber) }, startIndex: 1).Get()
-            );
+            ).ConfigureAwait(continueOnCapturedContext: false);
         }
 
         private async Task<ResponseResult<string, ResultErrorCode>> SendModificationInvoices(Receiver receiver, InvoiceNumber originalInvoiceNumber)
         {
-            var exchangeToken = await NavClient.GetExchangeTokenAsync();
+            var exchangeToken = await NavClient.GetExchangeTokenAsync().ConfigureAwait(continueOnCapturedContext: false);
             return await NavClient.SendModificationDocumentsAsync(
                 token: exchangeToken.SuccessResult,
                 invoices: Sequence.FromPreordered(new[] { CreateModificationInvoice(originalInvoiceNumber, receiver) }, startIndex: 1).Get()
-            );
+            ).ConfigureAwait(continueOnCapturedContext: false);
```

Comment:
Meh, this is starting to be a lot of code. Maybe let's rethink it and talk about it tomorrow if we actually want to keep targetting .net framework 😄 
cc @marektresnak

Reply:
Maybe we actually could. I don't think we have many users that are writing blocking code on the old platform (if any). So if we just release it as a new major version to make it clear there might be compatibility issues, it shouldn't really affect anyone.

My guess is that only the Czech library is actually being used by other users, and as far as I remember, we haven't done any significant changes in a long time (just some with the migration to .NET core). So not upgrading to the new version is still an option for anyone, who requires `ConfigureAwait`.